File: 04_python/aula_079_2/exercicio_02/Pessoa.py
class Pessoa:

  def __init__(self, nome, idade, morada):
    self.nome = nome
    self.idade = idade
    self.morada = morada

    # nova_pessoa = {
    #   "nome": nome,
    #   "idade": idade,
    #   "morada": morada
    # }
    # return nova_pessoa

  def toString(self):
    print(f"{self.nome} - (Idade: {self.idade}) [Morada: {self.morada}]")

File: 07_javascript/aula_144/exercicio_02/Pessoa.js
class Pessoa{

    // Atributos Privados
    #cartao //dessa mesma forma, precisa ser criado um atributo que seja privado

    // Construtor e Atributos
    constructor(nome, idade, morada){
        this.nome = nome;
        this.idade = idade;
        this.morada = morada;
        this.#cartao = 1111;
    }

    //Métodos
    complertarAnos(){
        this.idade++;
    }

    toString(){
        return `Nome: ${this.nome} | Idade: ${this.idade} | Morada: ${this.morada}.`;
    }

    novaMorada(nova_morada){
        this.morada = nova_morada;
    }

    casou(novo_apelido){
        this.nome += " " + novo_apelido;
    }

}